src/program.js

```diff
@@ -298,6 +298,12 @@ Example: $0 --help run.
         type: 'string',
         coerce: coerceCLICustomPreference,
       },
+      'url': {
```

Comment:
@saintsebastian how about renaming this new CLI option to something like `--open-url` or `--start-url`?